converter/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -0,0 +1,166 @@
+	REPORTING_PARAMETERS_MISSING_PERFORMANCE_YEAR("Must have a performance year"),
+	QUALITY_MEASURE_SECTION_REQUIRED_REPORTING_PARAM_REQUIREMENT("The Quality Measure Section must have only one Reporting Parameter ACT"),
+	PERFORMANCE_RATE_INVALID_VALUE("Must enter a valid Performance Rate value"),
+	CPC_CLINICAL_DOCUMENT_MISSING_PRACTICE_SITE_ADDRESS("Must contain a practice site address for CPC+ conversions"),
+	CPC_CLINICAL_DOCUMENT_ONLY_ONE_APM_ALLOWED("One and only one Alternative Payment Model (APM) Entity Identifier should be specified"),
+	CPC_CLINICAL_DOCUMENT_ONE_MEASURE_SECTION_REQUIRED("Must contain one Measure (eCQM) section"),
+	CPC_QUALITY_MEASURE_ID_INVALID_PERFORMANCE_RATE_COUNT("Must contain correct number of performance rate(s). Correct Number is %s", true),
+	NUMERATOR_DENOMINATOR_MISSING_CHILDREN("This %s Node does not have any child Nodes", true),
+	NUMERATOR_DENOMINATOR_INCORRECT_CHILD("This %s Node does not have an Aggregate Count Node", true),
+	NUMERATOR_DENOMINATOR_TOO_MANY_CHILDREN("This %s Node has too many child Nodes", true),
+	NUMERATOR_DENOMINATOR_MUST_BE_INTEGER("This %s Node Aggregate Value is not an integer", true),
+	NUMERATOR_DENOMINATOR_INVALID_VALUE("This %s Node Aggregate Value has an invalid value", true),
+	IA_SECTION_MISSING_IA_MEASURE("The IA Section must have at least one IA Measure"),
+	IA_SECTION_MISSING_REPORTING_PARAM("The IA Section must have one Reporting Parameter ACT"),
+	IA_SECTION_WRONG_CHILD("The IA Section must contain only measures and reporting parameter"),
+	NPI_TIN_COMBINATION_MISSING_CLINICAL_DOCUMENT("Clinical Document Node is required"),
+	NPI_TIN_COMBINATION_EXACTLY_ONE_DOCUMENT_ALLOWED("Only one Clinical Document Node is allowed"),
+	CPC_QUALITY_MEASURE_ID_MISSING_STRATA("Missing strata %s for %s measure (%s)", true),
+	CPC_QUALITY_MEASURE_ID_STRATA_MISMATCH("Amount of stratifications %d does not meet expectations %d for %s measure (%s). Expected strata: %s", true),
+	IA_MEASURE_INCORRECT_CHILDREN_COUNT("Measure performed must have exactly one child."),
+	IA_MEASURE_INVALID_TYPE("A single measure performed value is required and must be either a Y or an N."),
+	MEASURE_PERFORMED_MISSING_AGGREGATE_COUNT("Measure performed must have exactly one Aggregate Count."),
+	MEASURE_DATA_VALUE_NOT_INTEGER("Measure data must be a positive integer value"),
+	CPC_NPI_TIN_COMBINATION_MISSING_NPI_TIN_COMBINATION("Must have at least one NPI/TIN combination"),
+	CPC_PERFORMANCE_PERIOD_START_JAN12017("Must be 01/01/2017"),
+	CPC_PERFORMANCE_PERIOD_END_DEC312017("Must be 12/31/2017"),
+	QUALITY_MEASURE_ID_MISSING_SINGLE_MEASURE_POPULATION("The measure reference results must have a single measure population"),
+	QUALITY_MEASURE_ID_MISSING_SINGLE_MEASURE_TYPE("The measure reference results must have a single measure type"),
+	QUALITY_MEASURE_ID_INCORRECT_UUID("The eCQM (electronic measure id: %s) requires a %s with the correct UUID of %s", true),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 123 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)